### ISSUE TITLE
FinTS dumper without IBAN (e.g. credit cards)

### ISSUE THREAD

**Issue by moritzrupp** [schurig/ynab-bank-importer#15]
Hi @schurig,

I'm currently evaluating your ynab importer.
I was wondering if it is possible to also fetch transactions from credit cards account (e.g. from DKB)?
As far as I understand they are identified by IBAN, but the credit card accounts do not have a IBAN, only a account number.

Am I correct that this scenario is currently not implemented?

**Comment by schurig** [comment]
Hi @moritzrupp, I'm also having an account at DKB and will try it later this week to see if it works. The IBAN is only used to match internal transactions. What I do is in each transactions I check if the receiver or recipient of a transactions is one of your own bank accounts. If yes, then I give the transaction a color in ynab so you see it.

I remember that DKB works a bit differently than other banks like N26 - they have credit card transactions and around the 22nd of the month they create one big transaction on your Giro-Card account to get the credit card back to 0€. Personally, because of  that I stopped using the DKB credit card and started to only use N26 or ING Diba for credit card expenses.

**Comment by schurig** [comment]
You're right! If I get retrieve the transactions of the credit card account then I don't see any reason against also getting them. But maybe we should then exclude the monthly credit-card transaction from the Giro-account since it doesn't contain any valuable transaction information.

If it really doesn't work with `ruby_fints` maybe there is an other library that can handle the DKB credit card transactions, then we could write a dumper for DKB credit cards.
